config/ie/options

```diff
@@ -239,7 +239,7 @@ if targetAppVersion :
 	# avoid the proper dependency order for Houdini 16.5 and earlier.
 	# this is not ideal, but we're too far along in the H16.5 lifecycle
 	##\ todo: Remove this clause once we're onto H17.0 across the studio.
-	if targetApp == "houdini" and distutils.version.LooseVersion(targetAppVersion) < distutils.version.LooseVersion("17.0") :
+	if targetApp == "houdini" :
```

Comment:
It seems that the comment above might be inaccurate now?

Reply:
updated comment to better reflect the change